### ISSUE TITLE
There was a problem with the selected skills response

### ISSUE THREAD

**Issue by InterstellarScout** [joanaz/MMM-MirrorMirrorOnTheWall#24]
Greetings! 
I have built myself this magic mirror and absolutely love the idea of making this MirrorMirror module work. I really love the instructions as well. They remain fairly up to date and do exactly what they should without question. 

Now in the end, the module only half works. I can invoke the app in Alexa:
Me: Alexa, Open Magic Mirror
Alexa: Hello my queen...

but then every command I ask from there on responds with 
"There was a problem with the selected skills response"

I am using the new JSON code which was required because the original code in the instruction was dated with this version.

The major differences that I have noticed from the instructions was that my src archive (containing index.js and MirrorMirror.js) ended up zipping down to 7.3 MB instead of the instruction's image of 5.2. I don't think this may say anything because Lambda accepted it and allows the skill to be invoked. 

Basic Information:
Raspberry Pi 3B+
MagicMirror Version 2

Thanks in advance! 
I look forward to figuring this out.

**Comment by a-schutt** [comment]
Did you ever determine what was causing the error? I am hitting a roadblock at this exact spot as well.

**Comment by Centurion83** [comment]
I did, there are dependency issues I fixed with sudo npm install. I also
reworked the xml file and success

**Comment by Centurion83** [comment]
Fork the repository and let smartbot or whatever auto update everything

**Comment by Centurion83** [comment]
Or go into the config file and change the dependency version number .
There's a few ways

**Comment by niaz151** [comment]
Hi, I'm facing the exact issue at the moment. What exactly did you run ```npm install``` for and what files are you referring to?

**Comment by Centurion83** [comment]
Oh I altered some things, I'll post my fix when I'm off work.

**Comment by niaz151** [comment]
No rush and thanks for getting back to me. Looking forward to the solution

**Comment by Centurion83** [comment]
The issue I encountered was in the aws side originally and it had to do
with an outdated version of a package . I can't remember off the top of my
head which

**Comment by niaz151** [comment]
How does the ```npm install``` command tie into the AWS side of things?

**Comment by Centurion83** [comment]
Like I said I changed some things around and ran npm. I'm an hour out yet
I'll post my build

**Comment by niaz151** [comment]
Hey, any chance you can post your build when you're free

**Comment by Centurion83** [comment]
Yes I'm sorry I'm trying to get to it, I teach scuba so things are a little
hectic

**Comment by Centurion83** [comment]
I know off the top of my head aws was failing because of an outdated
package version. That part I fixed originally by manually installing the
version called for by Magic Mirror install script . Also you can change the
version number, and most importantly was getting the zip folder exactly
correct . I'll post today

**Comment by niaz151** [comment]
Hey just wanted to check in and see if you've managed to figure out the build, no rush though

**Comment by Centurion83** [comment]
Oh yeah actually I'll rebuild the build and post all my log.

**Comment by niaz151** [comment]
Sorry for constantly asking lol but any update?

**Comment by kizz91** [comment]
"1 year later"
